packages/fluentui/react-builder/src/components/ReaderNarration.tsx

```diff
@@ -66,7 +72,7 @@ export const ReaderNarration: React.FunctionComponent<ReaderNarrationProps> = ({
           // Preselect the path and the narration element to the first focusable element with tabindex >= 0
           const preselectedElementItem =
             focusableElementsItems.find(item => item.element.tabIndex >= 0) || focusableElementsItems[0];
-          selectedElementPath = preselectedElementItem.path;
+          selectedElementPath = preselectedElementItem.path.join(pathSeparator);
```

Comment:
Why is this change necessary ? was it not a string before ? 

Also what is the current type ? seems to me that it is `any` which makes reviewing this harder

Reply:
Okay, I  have added types for all the global variables in this file.